backend/src/main/java/com/aadeshandreas/ailearning/ai_learning_companion/model/Question.java

```diff
@@ -0,0 +1,15 @@
+package com.aadeshandreas.ailearning.ai_learning_companion.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@Setter
+@Getter
+public class Question {
+    private String question;
+    private List<AnswerOption> options;
+    private String answer;
```

Comment:
```suggestion
    private int answerId;
```
Can we make this an int so it just return the id of one of the AnswerOptions